install.sh

```diff
@@ -28,5 +28,8 @@ BDM_PROJECT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
 # include util functions
 . $BDM_PROJECT_DIR/util/installation/common/util.sh
 
+# Install all prerequisites
+$BDM_PROJECT_DIR/prerequisites.sh $(DetectOs) all
+
 # call install script for the detected OS
 CallOSSpecificScript $BDM_PROJECT_DIR install.sh
```

Comment:
Let's call `util/installation/common/install.sh` directly.
All os specific scripts forward the call 
Remove all os specific `install.sh` files